CMakeLists.txt

```diff
@@ -276,6 +276,10 @@ if(GIT_RESULT EQUAL "0")
     # not on a tag, increment the patch version by one and add a -devX suffix
     math(EXPR DUCKDB_PATCH_VERSION "${DUCKDB_PATCH_VERSION}+1")
     set(DUCKDB_VERSION "v${DUCKDB_MAJOR_VERSION}.${DUCKDB_MINOR_VERSION}.${DUCKDB_PATCH_VERSION}-dev${DUCKDB_DEV_ITERATION}")
+
+    if (NOT DUCKDB_DISTRIBUTION)
+      add_definitions(-DDUCKDB_DISTRIBUTION=\"dev\")
+    endif()
```

Comment:
I would put this code top-level or make sure it's defined also in other branches.

Reply:
Should it be "dev" or some other value like "unknown" or "manual"? I'd like to provide a default, and it would also be good to find any places I missed setting it in the CI.

Reply:
done - added to all branches